modules/client/examples/05-encoders-decoders/12-multisig/01-add-addresses.ts

```diff
@@ -6,20 +6,15 @@ title: Add Members
 ## Add Members from the Multisig Plugin
 
 Adds new address as members of the Multisig plugin installed in a DAO, so they are now able to vote on proposals.
+
```

Comment:
We dont have a findInterface example for multisig but we do have it for the other plugins